tests/runtime/basic

```diff
@@ -158,6 +158,22 @@ EXPECT hardware:cache-misses:
 EXPECT tracepoint:xdp:mem_connect
 TIMEOUT 5
 
+NAME it lists given search pattern
```

Comment:
I don't think we need this additional tests. The ones above it already cover this case.